code/modules/mob/mob.dm

```diff
@@ -45,13 +45,7 @@
 	ghostize()
 	if(mind?.current == src) //Let's just be safe yeah? This will occasionally be cleared, but not always. Can't do it with ghostize without changing behavior
 		mind.set_current(null)
-	QDEL_LIST(mob_spell_list)
-	for(var/datum/action/A as() in actions)
-		if(istype(A.target, /obj/effect/proc_holder))
-			A.Remove(src) // Mind's spells' actions should only be removed
-		else
-			qdel(A) // Other actions can be safely deleted
-	actions.Cut()
+	actions?.Cut()
```

Comment:
Is this what TG does? Nothing with the actions?

Reply:
Apparently they do in fact do nothing with the actions. I don't know if it actually matters though